_config.yml

```diff
@@ -0,0 +1,81 @@
+name: Fast Ruby Blog
+description: Ruby on Rails Upgrades
+meta_description: "It's about time you upgrade your Rails version"
+
+markdown: redcarpet
+highlighter: pygments
+logo: true
+paginate: 15
+baseurl: /blog
+domain_name: 'https://fastruby.io'
+permalink: none
+category_dir: 'tags'
+excerpt_separator: "<!--more-->"
+author_title_prefix: "Articles by "
+timezone: America/New_York
+
+# Details for the RSS feed generator
+url:            'https://fastruby.io/'
+author:         'OmbuLabs'
+feed:
+  path: rss.xml
+
+exclude: [vendor]
+plugins:
+  - jekyll-paginate
+  - jekyll-feed
+
+sass:
+  style: compressed
+
+defaults:
+  -
+    scope:
+      path: ""
+      type: posts
+    values:
+      is_post: true
+
+authors:
+  etagwerker:
+    display_name: Ernesto Tagwerker
+    gravatar: 9299203e7e9b9ae35e39eb66f3155f15
+    email: ernesto@ombulabs.com
+    twitter: etagwerker
+    github: etagwerker
+  mauro-oto:
+    display_name: Mauro Otonelli
+    gravatar: 350875eb8fce8553e12040c5a1f5d5d0
+    email: mauro@ombulabs.com
+    twitter: mauro_oto
+    github: mauro-oto
+  schmierkov:
+    display_name: Sirko Sittig
+    gravatar: ff0ffbb39659d36ad6aabcd0b4715c1b
+    email: sirko@ombulabs.com
+    twitter: schmierkov
+    github: schmierkov
+  cecilia:
+    display_name: Cecilia Farías
+    gravatar: a114aaf647736b09d7f09466fcaed51c
+    email: cecilia@ombulabs.com
+    twitter: xicadomar
+    github: cecilia-ombulabs
```

Comment:
@cleicar No need to list ex-employees who haven't written articles which will be copied/pasted in this blog.